brainzutils/mail.py

```diff
@@ -22,29 +23,35 @@ def send_mail(subject, text, recipients, attachments=None,
         from_name: Name of the sender.
         from_addr: Email address of the sender.
     """
-    if attachments is None:
```

Comment:
why'd you change the attachments code in the functiion, I only asked for the changes in the test, let's revert them back as the old way is a bit cleaner and pythonic. And I don't want to change code that is hard to test a lot.

Reply:
Cool :)